src/platform/imx8ulp/include/platform/drivers/interrupt.h

```diff
@@ -0,0 +1,79 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2019 NXP
+ *
+ * Author: Daniel Baluta <daniel.baluta@nxp.com>
+ */
+
+#ifdef __SOF_DRIVERS_INTERRUPT_H__
+
+#ifndef __PLATFORM_DRIVERS_INTERRUPT_H__
+#define __PLATFORM_DRIVERS_INTERRUPT_H__
+
+#include <sof/bit.h>
+
+
+/* IRQ numbers */
+#define IRQ_DSP_SYS_NMI      0
+#define IRQ_NUM_SOFTWARE0    1
+#define IRQ_NUM_TIMER0       2
+#define IRQ_NUM_TIMER1       3
+#define DSP_Profiling_IRQ_L3 4
+#define DSP_Wer_L3 5
+#define DMA2_c0_ErrTranc 6
```

Comment:
This seems out of place with the regular naming convention. If you aren't using or handling them from SOF, you don't need to define them at all anyway.